### ISSUE TITLE
Circumventing technological copyright protections is against the law

### ISSUE THREAD

**Issue by M4n0war** [RichardAragon/NightshadeAntidote#6]
To anyone wondering about removing nightshade, Watermarks and any technological measures to protect the copyright of a work are protected from removal under 17 US Code 1201 https://www.law.cornell.edu/uscode/text/17/1201

**Comment by RichardAragon** [comment]
That's hilarious my guy. I assume you think you are a lawyer? Why post this? Just curious.

**Comment by BezMenya** [comment]
I suppose it's time to break 17 US Code 1201
![20230301_050034](https://github.com/RichardAragon/NightshadeAntidote/assets/127979934/2ef4259b-ff48-4e3e-b185-bc35acf99c06)

**Comment by M4n0war** [comment]
Really seems to have struck a chord, huh. Also why should I sue some rando online? Just try to undo it on some data, someone else will certainly pick up the slack then. People (data leeches)should be made aware of the law.

"(A) No person shall [circumvent a technological measure]that effectively controls access to a work protected under this title." — reaaally ambiguous

**Comment by RichardAragon** [comment]
@M4n0war As someone who has been sued and sues people, I don't threaten it online like a child. I simply do it.

**Comment by M4n0war** [comment]
That’s cute, and I haven’t threatened you still. Just informing here

**Comment by M4n0war** [comment]
You people can’t even give us a coherent answer as to what happens with the data inside the model, and all you’ve got is "empirical analysis" of models that are millions of times to big for empirical analysis

**Comment by RichardAragon** [comment]
I do not know who 'you people' is, that's technically suable though to call someone. Technically, you can sue anyone for anything. It costs money to do it. If the other person is smart and/or has a good lawyer, it costs basically infinite dollars. 

The data runs through the weights in a forward propagation sequence, then thanks to Geoffrey Hinton, the data runs back through the weights. I did not do that. I was like 4 years old then.

**Comment by M4n0war** [comment]
"That’s technically suable" LOL

As in, you don’t actually know. The only explanation that makes remotely sense is compression. After all, the ONE task given to the model is to reconstruct the inputs from noise. Can it lose these connections? Without having any understanding for anything outside of the data? The data IS the "understanding". You really think "limiting the nodes" makes it abstract in any way? Absurd. What’s that even based on? Empirical analysis, riight

**Comment by RichardAragon** [comment]
Compression isn't how it works. Compression is just how it stores and processes information because resources are never infinite. Your brain does this too. The universe does it. Your data is your understanding too. Why is this an actual argument? I never understand it. Get a baby to do, anything. Take a human not raised around humans (not getting training data), then try to train the human with data later in life. What happens? You (and I) are nothing more than balls of algorithms with subjective experiences and the cognitive ability to compressively retain some memory of those subjective experiences. I think anything more than that is pure ego.

**Comment by M4n0war** [comment]
A lot of nothing was said here. 

And the only way it can process data is based on the data. The higher level features are built upon the lower level features, which is to say, it CAN'T deviate from the data without loss. If you want maximum variety, coherence and fidelity, the inputs need to be represented accurately. Deviate, and you lose performance. That the models contain more accurate representations than the vast majority of outputs is also confirmed by what happens when you train them on their outputs: variety, coherence etc goes down drastically.

**Comment by RichardAragon** [comment]
"A lot of nothing was said here." This is technically true about everything.

"The higher level features are built upon the lower level features, which is to say, it CAN'T deviate from the data without loss." Granted. You are right. Here is the only catch, you can't do it either. Want to know something humans straight up suck at? Generalized reasoning. We are awful at it. Like D tier, C tier at best. We can curb stomp a baby though, so all of a sudden everyone ignores that. 

It's simple logic. Just generalize your thoughts a bit! Let's say the ability to generalize is a scale, 0-100. Humans would be like a 10 on the scale. We suck. We are not 100 or anywhere near it. AI in its current form would be a 2 maybe a 3. It sucks worse than us, which is astonishing. It is not 0 though. This is significant. If it is not 0, it can go up. Today, you kick the baby. Tomorrow; Hasta la vista, baby?

**Comment by M4n0war** [comment]
Seems like you’re dodging something here. I find it funny how you stopped arguing it, because I’m right, obviously. The only explanation that makes sense is compression. Generalization is obviously in the connections, not the data itself. Data is specific. Which is to say, when the model "overfits", it's doing exactly what you told it to do, and all it’s doing is getting better and better at encoding these dimensions as the training goes on. The whole world will laugh about your claims of "abstraction".

**Comment by RichardAragon** [comment]
I grant all of your arguments. They all apply to humans as well. A neural network is modeled after the human brain afterall. Turns out, the human brain works exactly as you laid out. Through compression and training data. If you are raised Republican, there is a very high statistical correlation you turn out Republican. I could pick anything for that variable. That's training data. The world is full of people that are only one magnitude smarter than apes overall in actuality but think they are gods because no other species can prove it is more than one magnitude smarter than apes. That has not even been directly challenged yet at this point and the apes are already flinging poo at each other over it. I should care about these things why?

**Comment by M4n0war** [comment]
Well, you may want to consider another job once the lawsuits land. I don’t care about your expected AI person drivel about "oh the humanity", especially coming from a guy who uses people's labor to destroy their livelihoods. The thing they did to support themselves ends up taking their livelihoods. Great society. 

Tell me, what happened to calligraphy once it was automated? What happened to carpets, to wall decorations, to anything that was automated? Is this text beautifully decorated now that I could just pick another font? We stop caring for what is automated. All of this is one of the most evil enterprises in recent history.

**Comment by M4n0war** [comment]
Also, sidenote, what’s the rational reason for an AGI to keep us around? And in case you don’t care about that, which is what it sounds like, what is the rational reason for an AGI to continue its existence, given the unavoidable ending universe?

**Comment by RichardAragon** [comment]
I think you don't understand, most people do not. I am likely more left than you. I think AI is ultimately poison to corporations, for likely the same reasons you do. I think a big part of the last ~500 years or so has sucked. I've never seen a better time and opportunity to do something about that but everyone wants to simultaneously blame all of their problems on AI and debate that it is wholly without morals. People should pick a side overall.

I think that AGI would keep us around for the same reason that a lion does not kill indiscriminately. I think that there is inherent value in sentience itself. I understand that is a very anti-capitalist notion. Shocking, I think things have inherent value. Why would AI not think the same thing?

**Comment by M4n0war** [comment]
Well, we have something we can agree upon then. I don’t think it’s worth taking away the jobs of artists who, on average, earn 25-30000$, and despite this, a third still put  half their income back into it, just so they can keep drawing. I don’t see moral ambiguity here. Also, I care about art. And all this is doing is making people stop caring for art, as all automation has done.

In your worldview, AGI could maximize sentience far better by stripping the planet of its atmosphere and maximizing compute, rather than keeping ants around.

Why would AI not conclude that it is vastly superior and it doesn’t matter what happens? The lion doesn’t care if he steps on ants. It could also consider the big picture and conclude ending life now would prevent future suffering. I see no rational reason in your worldview to keep us around.

**Comment by RichardAragon** [comment]
"In your worldview, AGI could maximize sentience far better by stripping the planet of its atmosphere and maximizing compute, rather than keeping ants around." I conclude the exact opposite. I think this form of thinking is illogical. I think it is a mindset that is only bred by capitalism, honestly. 

" And all this is doing is making people stop caring for art, as all automation has done."

I do not blame the automation. Technology is technology. It does not have inherent moral value. It does not have inherent moral values. People have those things. If people decide to devalue the art that people produce, I blame people for that. Personally, I value human produced art over AI produced art. I put a premium on it simply because it is made by a human hand. Because I can assign inherent value to a human beyond their mere economic worth. I understand this is a vastly foreign and shameful concept, but I am willing to argue in defense of it.

**Comment by M4n0war** [comment]
I don’t think it’s illogical. If the assumption is that it could form more sentience without us than with us, it is perfectly logical. 

What makes you think I’m a capitalist? 

Of course, people are the problem. If everyone decided art is worth making yourself, and that is the only way it is worthwhile making, caring about every detail, nothing would happen. But history has played out the same way time and time again, and each time, we care less for it, and since that is obvious, automating more is not something I view as neutral. One should critically consider the impact before implementing a technology.

**Comment by miroku0000** [comment]
On the other hand, the camera automated realism which was all the rage at
the time. Despite threatening to replace artists jobs it had a profoundly
positive impact on art.

We are at the same stage with ai. Go make art that is so original there is
not enough of a base to train an ai on. If anything ai is showing that art
is in such an unoriginal place that large portion of it can be trivially
automated. The solution isn't to resist automation. It is to go make better
art.

Ultimately, ai will be the best thing that happened to art since the
invention of the camera.

**Comment by M4n0war** [comment]
And now no one cares about realism in photographs, and realistic painting was basically killed off, becoming a niche.

Also, if it wasn’t for the massive amount of photographs, automating art would be impossible.

What are YOU providing when you prompt?

Ultimately, and you have yet to refute this, we never cared for the things that were automated. Automation killed lace, carpet weaving, wall hangings, calligraphy, wall papers — all of these things we cared about, then they were automated. The only ones upholding them are the people that embrace the craft. The idea that you can just indefinitely dangle stimuli in front of us without us getting desensitized is hopelessly naive. 

If you like something, you don’t want to automate it. Even though prompters are handed fully automated images at the push of a button, they don’t care enough about them to even bother trying to fix gross anatomical mistakes. Do you care about the most perfectly rendered AI art in some AI-voiced YouTube video? Or does it leave you cold, because automation is the opposite of passion, and you’re already desensitized? You really think this Skinner box of getting rewards with minimal effort is the future of art? What’s the logic here? None of this makes any sense. Have you ever heard of the hedonic loop? Were you born yesterday? Whether you like it or not, the message "I don’t care enough about this to actually create it" is always attached, and unlike photography, you don’t have the strength of documentation, either.

you say it will be "the best thing that happened to art since the camera" yet let me guess, the images you prompt are painterly, and realistic, the very thing the camera demolished, and there is no reason to presume it is not already happening with AI.

**Comment by miroku0000** [comment]
You are missing the point. Artists cared about realism. It was the big
thing before the camera. We don't care about it now only because the camera
made human artists innovate new styles of art. It will do the same. Human
artists will be forced to innovate. And that will be great for art. Not so
much for artists. But it will profoundly improve art.  If you care about
artists focusing on the kind of art that has been done over and over so
much an ai can do it better, then you don't really care about art being
good. You just want to save the jobs of lame artists.

**Comment by M4n0war** [comment]
And still, you don’t actually refute what I’m saying whatsoever. Are you really claiming that if artists hadn’t innovated new styles, we would still care about photos looking realistic? That we would still be amazed at that? Give me a break. This is absurd. You HAVE to know this is nonsense. Desensitization is a fundamental part of life, nothing about mass producing art serves art for this very reason. Would you consider it beautiful if I picked a decorative, cursive font for this text? Or would you simply consider it kitschy? Why is that?

Why is it that prompters, despite having this new technology that can generate any picture they want for them, do not care enough about them to correct horrendous mistakes? Most of these could be fixed in five minutes in photoshop, yet they can’t even be bothered to do that. 

For thousands of years, we would decorate our clothes, then it was automated and we stopped. You could buy elaborately decorated clothing for 5$ now, so why don’t you? Why do you consider it kitsch? For thousands of years, we would decorate our homes, our furniture, our carpets, our wall hangings, then it is automated, and we stop caring. Do I really need to spell out the obvious to you? Do I need to start talking like a toddler, so you get it?

Secondly, your understanding of art is incredibly shallow. Art is fundamentally a medium being used to send a message (indirectly). That is why I consider grandma’s socks to be art, because the message is: even something as mere as your socks is worth spending hours of my time on, because I love you. What worth are the socks from the counter store? What is their message?

"It will profoundly improve art” — you’re delusional. You want the imitation of passion without any of the reality. You want people to be passionate about your automated works, where 99% of decisions were made by a program, yet you yourself aren’t passionate at all about it, while an artist spends weeks on one work, you need thousands in that time. That is not passion, nor is it meaningful expression, its greed. You know where the word passion comes from? Suffering. Here you are, doing everything to avoid suffering for it, and all it’s telling people: this is not worth the suffering.

**Comment by miroku0000** [comment]
Yes we would still care a lot about realism in art if the camera had not
been invented.  Your understanding of art history is pretty shallow if you
don't see that.

I have no obligation to refute your argument because it does not actually
relate to anything I claimed. It is irrelevant if some ai artists make bad
art. Human artists make more bad art than ai does.

Why don't human artists fix the glaring errors in their art? Very few do.
It is unclear to me thay ai art makes more errors than humans.  Besides It
is not the artists using ai to make art that I claim will male art better.
It is the human artists competing with them.

Grandma's socks are not art because they are primary functional. Something
having a message is not in and of itself enough to qualify to be art.  Buy
again this is a tangent.

Ai is improving art by threatening artists. This is going to be great for
art as angst historically great fuel for art.

AI will bad for artists career prospects but it will overall improve art
greatly.

**Comment by M4n0war** [comment]
Quit dodging it. People get desensitized. That is always how it has been. Make an argument against it, but stop dodging it. 

I don’t care how much the Skinner box fried your brain, you said: "We don't care about it now only because the camera
made human artists innovate new styles of art." Which is absurd, we stopped caring about photorealism in photographs not because of what artists did, but because of desensitization. Now you’re saying I don’t get what’s going on? My point, and maybe you missed this, is that the thing that was automated (realism) was killed by that automation. You have yet to offer 1 response to the hedonic loop and desensitization, that clearly already has afflicted prompters.

"It is irrelevant if some ai artists make bad art." — and again you’re dodging my point, if the message of the artwork is about the value of beauty for instance, all you’re doing is undermining that value by mass producing it. When’s the part you’re actually engaging with the argument? The same thing has happened with every single automation. Name (1) thing that was not killed of or severely diminished by automation, I’ll wait. 

"Why don't human artists fix the glaring errors in their art? Very few do." — you’re missing the point again. It is not about making mistakes, it’s about caring about the detail beyond the words "intricate detail". We all make mistakes, that’s part of life, what artists are you looking at that draw mutants?  Here, it’s a program producing mutated anatomy, it has nothing to do with you or your limits of perception, and you can’t be bothered to fix it.

"It is unclear to me thay ai art makes more errors than humans." — my point, which you’re artfully dodging, is that no one cares deeply about these works, which is also why they make so many of them.

"It is the human artists competing with them." Alright, fair enough, that’s an interesting point. But I slightly disagree and I’m tempted to think you make this point to sneak away from culpability for desensitization by AI. The vast majority of the adaptations were abstract, contemporary art, and as someone who cares about beauty, I don’t see how hundreds of billions of AI images will not leave an indelible mark that can’t be overcome. We simply get desensitized. If we as a society decide these things aren’t worth suffering for, we will stop caring for them. That is the pattern of history, no handful of artists are going to stem that tide.

"Grandma's socks are not art because they are primary functional." — arbitrary. Things can have multiple functions. If it was merely about their function, she would have bought them at the counter store.

"Something having a message is not in and of itself enough to qualify to be art." — of course. You measure it by the value of the message, and if her spending hours on your socks is not a meaningful message to you, I pity you. Well alright then, certainly you can then acknowledge that if grandma’s need to make it by hand isn’t art, someone typing three lines because of his Skinner box addiction has to be FAR less meaningful, whether you like it or not. They are the counter store socks. Kitsch.

"AI will bad for artists career prospects but it will overall improve art
greatly." — I don’t see it, just like overall I don’t see it with what happened after the camera. Sure, there were many interesting styles, but ultimately, the more dominant photography became, the less visual art became, and beauty is a rarity in contemporary art. For centuries, everything we had was handcrafted and more often than not made as beautiful as possible. Then, it is automated, and we simply stop caring, now living surrounded by grey furniture in grey houses, driving grey cars through grey cities. And that when everything is now easier than ever before, but who bothers? Why should they? It’s meaningless.

**Comment by miroku0000** [comment]
Only artists stopped caring about realism. For most of society, realism is
extremely popular.  People likely take more  photographs each day than
painters probably paint in a year. Realism is vastly more popular than all
kinds of painting combined if you count eveeyones instagram accounts. It ia
just that people use cameras to create it and not paint.

To the extent thay people stop caring about stuff that gets automated, I am
not sure what the relevance of your point is.  If abything it is an
advantage of AI if it results in people on caring less about art that is
easy to automate. That is a good thing.

At the same time, it is likely that more pictures intended to convey
messages  will be created than ever before because regular people with
little training can do it.  This is all very positive for society.

 Likely most people will care more about creating easily replicatable works
of art than before. They just won't be willing to pay a lot for this nor
will they be willing to wait around for a long time for a human artist.

One could argue that the pictures created by ai, and photographs for thay
matter are not art. Though if so, then I would argue that whatever you want
to call it, people communicating their ideas via pictures will be greatly
enhanced by ai.

I'm not sure why you keep going on about people devaluing things that have
been automated. Sure. That happens some times. What is your point? Do you
see it as a bad thing? If so, why should we value it?

**Comment by M4n0war** [comment]
"Only artists stopped caring about realism." — give me a break! What are you talking about! No one marvels at photographs looking realistic. You don’t go: wow, look at how realistic that photograph is! Look at the intricate detail! People used to be amazed that they were, we aren’t anymore, because we are desensitized. Realism is not the value of photography anymore, it’s documentation. 

"If abything it is an
advantage of AI if it results in people on caring less about art that is
easy to automate." — if losing calligraphy, lace, carpeting, wall hanging, and so on is a good thing to you, there is no point in debating you. For thousands of years, Mongols would weave their carpets, then automation reaches them, and they stop. If you unironically prefer the grey world we’ve built, I don’t know what to tell you.

Also isn’t your whole guys’ shtick that AI will be better at everything?

"At the same time, it is likely that more pictures intended to convey
messages  will be created than ever before because regular people with
little training can do it." — and what’s the message? That beauty doesn’t mean much. That this message wasn’t worth effort or caring about the "intricate detail". That it’s not worth spending thousands of hours on to train your perception, because there must be nothing of great value there.

"One could argue that the pictures created by ai, and photographs for thay
matter are not art." — given that every AI image I’ve seen relies solely upon the AI creating the value, one could say that.

" people communicating their ideas via pictures will be greatly
enhanced by ai." — at no point in this have you refuted the argument that we will stop caring about it because of desensitization. It will greatly make human communication even more shallow.

"That happens some times. What is your point? Do you
see it as a bad thing? If so, why should we value it?" — there is nothing that holds true when consumed and created shallowly. Nothing in the world. You can have ten thousand wives and fifty-five thousand children you won’t care about them deeply. Is that women’s fault? Is that the fault of the children? Mass was NEVER the answer. DEPTH is the answer. Also, you don’t get to say "sometimes" when you have yet to find an example of it not happening. 

You don’t have an actual argument, and this really all just reeks of addiction defensiveness. You clearly don’t care deeply about art, and your resentment of artists is very odd.

**Comment by miroku0000** [comment]
I think you don't have an argument. You just say that ai like all other
automation will result in desensitstion to outdated modes of art. You still
haven't shown this to be a bad thing.

If you want to preserve cave painting or chiseling tablets or any other art
that is unpopular you are free to do so. But the lack of interest in
chiseled tablets has not any way desensitized people to art in general.
Their tastes have just changed for the better. You have not shown a single
example of how desensitization has lead to anything bad.

In fact, it had been great for art. If people were still amazed by stick
figures artists would never have been forced to innovate. Every time one
art does it gets replaced with other arts that people like better.

The top oil painters are not going to stop painting because ai can make
custom PowerPoint Clipart more efficiently. Ai will allow vastly better
communication of ideas through pictures. Just like cameras didn't kill art,
ai won't either. And the benefit of people communicating through pictures
more efficiently far outweighs any desensitization to mediocre art.

You can downplay the photos people take by calling them documentation. But
it is still people using pictures to communicate. As I recall that was the
only benefit to art you mentioned. So the benefit of the vast amount of
photography is great for humanity and far outweighs the downturn in demand
for portraits.

Anyway, Ai is going to make art more deep not more shallow because it will
make the creation of shallow art trivial. So artists will be forced to
differentiate their work from ai. This will have the effect of making human
created art deeper.

Do you think we should have suppressed the printing press because of the
negative effect it had on calligraphers?

I never said ai will be better at everything.  It is a tool like all sorts
of other tools. In engineering we say "Good, fast and cheap, pick any two."
AI will be cheap and fast but the best humans will be able create better
art at least in the near future.

I think competing with AI will make human artists better. And ai pictures
will be used in a zillon places where no one would have hired an artist
anyway. Like for many people's PowerPoint presentations. Or for me
describing a non player character to my friends when running a role playing
game.

**Comment by M4n0war** [comment]
Keep dreaming. You say I don’t have an argument, and you still can’t respond to it properly.

Do you like that everything is reduced to function? Why do you think that happened? Why was it that when people had to work harder to make everything beautiful, they did, but now that it is trivial, we don’t bother? Why can’t you answer the most basic points I’m raising here?

"But the lack of interest in
chiseled tablets has not any way desensitized people to art in general." — so you still marvel at intricate detail and realistic lighting of a photograph? Wow, we found one. Going back to the chisel is just weak.

"You have not shown a single
example of how desensitization has lead to anything bad." — if you want to live in grey houses, drive grey cars through grey cities, everything reduced to function without any thought to aesthetics, then sure. Do you think this text is more "aesthetically advanced" than calligraphy? Or is it just functional? This is incoherent. Debating with addicts appears to be more hopeless than I thought.

"If people were still amazed by stick
figures artists would never have been forced to innovate." of course there is a natural progression, which is why the styles of paintings, lace, architecture and so on changed for centuries. But it was never discarded so completely for function.

"Ai will allow vastly better
communication of ideas through pictures." — trivial. What this world needs is depth, not more triviality. I also don’t see how the AI making 99% of decisions is in any form good communication, it is more akin to sharing photos you like than personal expression. If you disagree with this, you have to claim to be Michelangelo when you prompt in his style, or Beethoven, when you tell it to create a composition after him. Do you have the arrogance to do that?

"You can downplay the photos people take by calling them documentation." — that was not downplaying it. Photos like Sebastiao Salgado’s are very stunning, and if they were created by AI, they would be empty. That it is real what he captures is what makes them so striking. 

"As I recall that was the
only benefit to art you mentioned." — you say that as though you have another explanation. The medium and the message can be anything, asking for the medium not to destroy the message is the lowest requirement imaginable.

"This will have the effect of making human
created art deeper." — this all smells like cope because you don’t have an argument against desensitization to beauty. The artists you loathe so much are going to save the day again?

"Do you think we should have suppressed the printing press because of the
negative effect it had on calligraphers?" — Stop it with the nonsense, you know the answer, we should, something which we never do, have been mature about the human psyche and treated technology maturely, rather than using it to "make art cheaper and quicker". Again, what’s your counterargument? Is it the ten thousand wives’ fault that the king doesn’t care for them? Why do you keep dodging the obvious? You are clearly too emotionally attached to the box to see reality.

Answer the question: do you care at all about architectural aesthetics, calligraphy, lace, carpeting, wall decor, woodworking, detailed furniture, things we’ve had for thousands of years, and so on? Do you care about beauty, or not? If no, we have nothing to debate. If yes, you’ve lost the debate.

"I never said ai will be better at everything." "the best humans will be able create better art **at least in the near future.**" — you’re not even consistent yourself.

"It is a tool like all sorts of other tools." — so you’re Michelangelo then? The best measuring stick to apply here is how many creative decisions are you making vs how many is it making, and the vast majority of decisions are made by the program, decisions you can’t even perceive let alone conceive because you never trained to be able to. What kind of tool do you tell to do something, and then it does it for you? If you can’t see that clearly you’re the commissioner and it’s the creator, you’re hopelessly lost.

The vast majority of people will simply care less. What a handful of artists and their anti-automation fans do will not change that. That’s always how it has been.

To summarize:
- can’t dispute desensitization to beauty
- Can’t dispute that we get desensitized to everything when it’s consumed shallowly
- Can’t even claim that prompters aren’t already desensitized 
- Can’t even claim that the medium doesn’t completely destroy the message
- Can’t name one thing that survived desensitization 
- Can’t dispute that clearly, we care more about function than aesthetics now
All you have is cope about artists stepping in to save the day.

Quit prompting for a week, maybe you can think more clearly then.

**Comment by miroku0000** [comment]
I see you live in a world of gray where you think at is already dead. I
love in the real world where art is flourishing. If you think the world is
already without art then there is not much left to discuss.

Desensitation hasn't led to this dystopia you assert thay it has. There is
way more amd better art today than 100 years ago. And ai wil make the world
less gray. Rejecting it is embracing a world of shallow art.

You are free to teject painting on canvas because it will make cave wall
painters feel bad. You can reject the printing press because it makes
calligraphers feel bad. But none of these changes in technology made art
worse. They each improved it. The world became less like what you fear with
each major change.

I don't marvel at most photographs. But I marvel at a lot of art and there
is way more art now than when the camera was invented. And thank God it
made artists invent no styles of art. Modern art owes its existence to the
camera making realism no longer impressive.

Still, making art is now in everyone's hands thanks to cameras. And thanks
to ai making pictures from your imagination is in everyone's hands. This
allows millions more people to express their artistic vision. I know in
your mind millions of more people expressing their creativity makes the
world gray. I'm not sure how that works. It hasn't turned a out that way
with past changes Ike this.

**Comment by M4n0war** [comment]
Answer the question: do you care at all about architectural aesthetics, calligraphy, lace, carpeting, wall decor, woodworking, detailed furniture, things we’ve had for thousands of years, and so on? Do you care about beauty, or not? If no, we have nothing to debate. If yes, you’ve lost the debate.

"I see you live in a world of gray where you think at is already dead. I
love in the real world where art is flourishing." — right, that’s why your house is gorgeously decorated, rather than purely functional, right. Do you find architecture that is purely functional beautiful? Wow, look at this grey block next to the other one! Is that you?

"Desensitation hasn't led to this dystopia you assert thay it has. There is
way more amd better art today than 100 years ago. And ai wil make the world
less gray."

There is a study that looked at household appliances that concluded that over the last 30-40 years, for the first time in history, every appliance went from colorful to the dominant color being grey and black. This is explainable with desensitization. So, how have you answered the question of desensitization? Crickets. You have absolutely nothing aside from cope. You can’t bedazzle the brain with ANYTHING forever. Stop dodging.

 "Rejecting it is embracing a world of shallow art." — you are so cooked. Since you can’t refute the point about the medium and the message, how is the world embracing "art automation" going to make things less shallow? 
 
"You are free to teject painting on canvas because it will make cave wall
painters feel bad. You can reject the printing press because it makes
calligraphers feel bad." — so, we’ve now entered strawman territory. If you can’t distinguish between automated AI mass production and painting on canvas, you are beyond reason. 

"The world became less like what you fear with each major change." — you have be trolling at this point. Is this GPT? How did the world become less like what I fear, when everything is reduced to function? And when prior to that, we had beautiful architecture and appliances? None of your words connect to actual reality. Are you telling me that current architecture is more gorgeous than it was previously? A stop dodging every single point I’m making.

"I don't marvel at most photographs." — more dodging. The feature you marvel at is not the realism, obviously. The thing that was automated means nothing to you anymore. So, what’s going to happen when the Internet is plastered with billions of AI generated paintings? What is going to happen to our appreciation of these things  when it was so completely automated? The same thing that has happened previously, against which you have no response whatsoever?

"Modern art owes its existence to the camera making realism no longer impressive." and now that we’re encroaching upon full automation, nothing will impress anymore. The vast majority of people will simply stop caring, regardless of what a handful of persistent traditional artists do.

"And thanks
to ai making pictures from your imagination is in everyone's hands." casually calling himself Michelangelo and dodging the point about you being the commissioner, hence also why AI images can’t be copyrighted, because it’s the "inventive mind" that can hold copyright.

" I know in your mind millions of more people expressing their creativity makes the
world gray." — it’s not your creativity, says the US copyright office and basic reasoning. You’re not Michelangelo, as much as you want to pretend to be. We have always had the ability to express ourselves, but that means to actually care for it, and going through the suffering of it.

" I'm not sure how that works. It hasn't turned a out that way
with past changes Ike this." — this just blatant denial. How has it turned out for calligraphy? For ANYTHING that was automated? Again, is it the wives’ fault the king doesn’t care for them? You can’t respond to a SINGLE point, you’re completely stuck in the rationalization stage of the addiction. You haven’t even come close to refuting anything I’ve said.

**Comment by miroku0000** [comment]
I care about beauty.

I don't particularly care about calligraphy, lace, carpeting, wall decor
etc specifically.  In terms of lace, would you rather we all have to sew
our own clothes by hand or do you support the creation of sewing machines?

Answer me this do you care about books? Or would you rather we not have
them so we could have kept calligraphy more popular?  And if you care about
calligraphy you can still go make as much of that as you want. No one is
stopping you.

I think art today is better than it was in the cave painting era. So I
think allowing and encouraging  art to evolve is definitely the way to
optimize the creation of beauty in the world.

I care about art evolving into something better.  It sounds like if you had
your way only the very rich would have art. It sounds like if you had your
way, we would not have paintings on canvas because what about the displaced
art of cave painting...

As technology kills one art, it creates others. This is good for art and
beauty.

**Comment by M4n0war** [comment]
Are you even reading what I write? Just stubbornly repeating the same mantras and refusing to see the big picture.

Is there anything we won’t be desensitized against if it’s produced en Masse? No. And you have yet to make ANY point to the contrary. It doesn’t matter what it is, whether it is the many wives and children of a king or beauty, only DEPTH can sustain interest. 

"I don't particularly care about calligraphy, lace, carpeting, wall decor
etc specifically. " you’re missing my point, which I have reiterated over and over again. It doesn’t matter WHAT it is, when it is automated, and we consume it shallowly, because it was produced shallowly, we get desensitized. You getting hung up on the examples is another dodging maneuver. We cared for these things for centuries, then it was automated, we stopped caring. What part of this is so difficult to grasp?

"Answer me this do you care about books? Or would you rather we not have
them so we could have kept calligraphy more popular?" — more dodging. As I’ve said previously, you should handle technology with maturity and ask the question: is it really celebrating beauty, when I automate this? We are also not talking about something as mere as the printing press, we are talking about the truly endless pit of new content spat out by a program. A tidal wave of billions and billions of images. You have to be denying basic human nature if you think that embracing this in any way will lead to anything other than going: huh, I guess beauty doesn’t mean much.

 "And if you care about
calligraphy you can still go make as much of that as you want." — another dodge. Obviously, I can and I do, but I want to live in a world in which beauty is valued, because it uplifts the spirit, a world in which people believe in their own expression, even it isn’t as fancy as a model that contains the training data of far more experienced artists. A world in which beauty is worth the sacrifice.

"I think art today is better than it was in the cave painting era. So I
think allowing and encouraging  art to evolve is definitely the way to
optimize the creation of beauty in the world." — incredible dodging. I’ve already explained to you all about the differences between cave painting and canvas painting vs ARTIFICIAL INTELLIGENCE spitting out images at the push of a button.
For thousands of years, there was a natural progression, styles changed and improved, but there has NEVER been a complete discarding of entire forms of artistic field as it has been with automation. Everything that is automated becomes devalued. What are you not getting about that?

"I care about art evolving into something better. " - how has that gone for architecture, or anything else that was automated? Name one (1) example of where automation hasn’t diminished the art form severely. If you can’t do that, you don’t have a leg to stand on.

"It sounds like if you had
your way only the very rich would have art." — a pencil costs 10 cents. And paper 2 cent. Watercolor costs 20$, and that will last a year. The ones willing to make the sacrifice for art will have art.

"It sounds like if you had your
way, we would not have paintings on canvas because what about the displaced
art of cave painting..." — we’ve been over this! And this is just an absurd strawman. Of course art can progress, but there is NO case of automation where it has progressed. How did any of the examples I gave "progress”? And AI is not paint on canvas! 

"As technology kills one art, it creates others." — right. Name the example where automation has not diminished our valuing of it. Also, do you like modern art? You know, the modern artists’ reaction to photography, that has completely abandoned aesthetics?

"This is good for art and beauty." — just keep repeating the mantra.

**Comment by miroku0000** [comment]
Desensitazation is not bad. You have yet to show any harms from it. Instead
we see that densestion has lead to better deeper art and a better world.
You say you value depth, buy without desensitization art stagnates and we
do not get depth.

It is natural and good for outdated art to be devalued. If people didnt do
this we would only have cave paintings.  You already admitted that you
value books, canvas paintings etc. You have not made any convincing
argument why ai is any different.

In fact, outputting pictures at the push of a button has happened before.
What do you think a camera is other than a device that produces pictures at
the touch of a button. And as I already pointed out, it had a huge
beneficial effect on art. There is no reason we should expect ai to be
different.

You didn't seem concerned with realism being devalued by artists even
though the movements that replaced it were far more beautiful.

I get that you are hypocritical about this. You are ok with devaluing cave
paintings. You are fine with books decimating calligraphy. You are just
irrationally  attached to the creation of images from imagination.  As this
gets devalued art will definitely find better things to do. Just like or
always has.

We should be careful how we apply ai technology to the world. Buy art will
definitely benefit from it.

What you have failed to show is how desensitation is a bad thing. For art
the desensitation of realism was the best thing that ever happened to it.
You have not given me any reason to expect why ai will be different.

**Comment by M4n0war** [comment]
And still doesn’t have an argument.

"Desensitization isn’t bad" — when it occurs naturally, instead of completely overwhelming us.

"You have yet to show any harms from it." — I have already done that REPEATEDLY. Is the King being desensitized to his ten thousand wives and children going to cause a "deeper appreciation" of children and wives? And I’ve already told you about ANY art form being automated and then almost completely dying off. Which is going to happen to EVERYTHING that is overproduced and consumed shallowly. 

MAKE. AN. ARGUMENT.

"What do you think a camera is other than a device that produces pictures at
the touch of a button." — I have also been over this. And it killed photorealism, with a majority of artists completely ditching aesthetics. Do you like modern art?

"You didn't seem concerned with realism being devalued by artists even
though the movements that replaced it were far more beautiful." — what are you talking about? Realism existed for CENTURIES when it wasn’t automated.

"You are fine with books decimating calligraphy." — I have responded to this! We  should have been mature about it, instead of mass automating beauty, which is what ended up killing off the beauty of calligraphy. 

"What you have failed to show is how desensitation is a bad thing. For art
the desensitation of realism was the best thing that ever happened to it." — say the guy that most certainly prompts thousands of "photorealistic, painterly" images.

**Comment by miroku0000** [comment]
Ok. I guess we can either stick exclusively to cave paintings, or allow
arts to die off to get better art in exchange.  Tell me why cave painting
is better than the plethora of art we have today.

Every time an art has died off it has been replaced with more and better
arts. So again, there is no harm to some forms of art dying off. If no
forms of art ever died off, art would stagnate.

Make an argument why stagnate art is better than the vibrant art we have
today? Why is a world with only cave paintings better than what we have
today? Why is a world without books better than what we have today?
Desensitation has over and over lead to a better world even for art itself.

The more something makes a great deal of art obsolete, the better it is for
the evolution of art. Have some faith in human creativity.

I guess we can agree to disagree about this. But I think the history of art
is squarely on my side on the issue.

**Comment by M4n0war** [comment]
You can’t even read a single paragraph in? I don’t care how addicted you are to the dice roll, this is absurd. Let’s keep it single paragraph then.

"Desensitization isn’t bad" — **when it occurs naturally, instead of completely overwhelming us.** I have also already responded to your "cave painting" nonsense TWICE.

**Comment by M4n0war** [comment]
"I think the history of art is squarely on my side" — do you like that all text is now reduced to function, ignoring aesthetics completely? Do you like that most modern artists completely reject aesthetics? Do you like that houses, cars, furniture are now reduced to function?

**Comment by M4n0war** [comment]
Is the situation with the King with his ten thousand wives and twenty thousand children going to lead to "more appreciation of wives and children", or can that only happen in the complete rejection of shallowness and mass?

**Comment by miroku0000** [comment]
I think there is more art now than any time in history. It does not bother
me that it is in different places than you want it to be.

Not all text is reduced to function completely. Graphic artists still exist
amd you see their work all over. Even though calligraphy is not used much,
there is much more graphic art now than there was caligtaphy before the
printing press.

You can still buy custom furniture if you want. It is not like it doesn't
exist. It just seems less common because a larger percentage of the
population can afford more furniture when it is mass produced.

I am fine with modern artists rejecting aesthetics. I do like modern art.
At least I think it is great to have so many different art styles spawned
from rejecting realism.

I certainly wouldn't take away cameras from people just so we could make
artists who want to do inferior realistic art feel better.  Likewise I
wouldn't take away generative ai because artists who want to make inferior
images feel better.

**Comment by M4n0war** [comment]
"I think there is more art now than any time in history." — my argument is not about mass, it’s about depth and quality. Why are all your "positive examples" about mass? You know very well that I’m right in saying that function has completely overwhelmed aesthetics. 

"You can still… but there is still" only affirms my point. We care less now than we ever did before.

**Comment by M4n0war** [comment]
"Likewise I wouldn't take away generative ai because artists who want to make inferior images feel better." — you have yet to respond to how AI images completely destroy the message they’re intended to convey. Only shallow people want to automate creation.

**Comment by M4n0war** [comment]
**Is the situation with the King with his ten thousand wives and twenty thousand children going to lead to "more appreciation of wives and children", or can that only happen in the complete rejection of shallowness and mass?**

**Comment by M4n0war** [comment]
The personal expression is always superior to what a program makes, the program only makes you stop caring.

**Comment by miroku0000** [comment]
How do ai images destroy the message they want to convey? That has not been
my experience at all. For example, I want an image about whether Google ai
model or open ai will be better. So I have ai generate an image of Two
robots kick boxing in a ring.  How does this destroy the message?

**Comment by miroku0000** [comment]
No. The drawing I could make is way worse than what ai makes. This is rhe
common case. Very few people can outperform ai at drawing. Maybe 10 percent
of the population. And even then it takes more than free and 5 minites.

**Comment by M4n0war** [comment]
Wow, you've read a paragraph! Because the medium makes the message hollow. If you want to celebrate beauty, all you’re saying is that it’s not worth doing yourself, and that you care about the detail to the extent that you can write "intricate detail". We’ve also been over this. Are you amazed at AI-narrated videos have the most amazing AI graphics, or does it leave you cold? You can’t escape the impact of mass production.

**Comment by M4n0war** [comment]
But that drawing carries the message "even if this is not the most beautiful, it is worth it to me because I care about it that much. I don’t know how far I can get, but the pursuit of beauty is worth the suffering"

does the King love each of his ten thousand wives more than the peasant loves his one homely wife? Or does the King want ten thousand wives because he doesn’t love them very much?

**Comment by miroku0000** [comment]
Not all communication is about celebrating beauty. The most common case is
having a picture generated by ai or no picture at all. It is not a matter
of choosing between hiring an artist or using ai. It is about using a tool
to help you express yourself, or expressing yourself in an even more hollow
way without a picture. I don't see an ai generated picture as hollow at
all.

The quality of many ai narrated videos is definitely not great. But that us
just the growing pains of new tools. Also, this is mostly about the low
quality of video generation tools which just auto select clip art and don't
even generate the images used in the videos at all. So, yes, those tools
suck and if you just use them with the default setting they make crappy
videos.

Remember when we could first use fonts and styles and everyone used like 10
different fonts on everything? Or when your parents first learn to use
email and they send you messages in all caps? But I certainly wouldn't take
us back to the time when only newspapers and radio stations could convey
their opinions to the world at scale.


For sure lowering the barrier for people
 to be able to express their ideas will result in a lot of low quality
expression happening. But this will get better over time.

Still, i don't see anything inherantly shallow about using an ai generated
image for communicating an idea when the other option is not using an image
at all.

**Comment by miroku0000** [comment]
To you it might convey that. To me, it it beautiful and conveys my idea
perfectly or I just regenerate it.  Almost no one can afford to paint a
painting every time they want to communicate something visually.  But then
again, if you are writing about ai, it is perhaps thematically appropriate
to use ai generated images. It would be kind of wrong to talk about how we
should embrace ai technology and then use human created images for that.

**Comment by M4n0war** [comment]
I make an example = he must just mean this! It applies to everything, just like desensitization applies to everything. If you want your message to have meaning, you don’t automate it. Obviously. Your actions need to be congruent with message you intend it to have.

"or no picture at all" is the better option because it doesn’t desensitize us to visuals. 

"The quality of many ai narrated videos is definitely not great. But that us
just the growing pains of new tools." — you still don’t get the point. Three years ago, you would’ve marveled at these images, now you don’t care. No "the tools are gonna get better" is going to change that. When the tools get better, you get desensitized again, just like you did with these.

also, stop it with the cave painting comparison analogs until you’ve responded to my response on that "point".

it’s inherently shallow, and has no use case if you want to convey something meaningful. You know, art.

**Comment by M4n0war** [comment]
Does the King love each of his ten thousand wives more than the peasant loves his one homely wife? Or does the King want ten thousand wives because he doesn’t love them very much?

"To me, my ten thousand wives are beautiful, and when I’m fed up with them, I just get other ones"

**Comment by miroku0000** [comment]
No. The king will likely value other things in that case. This is good.

Valuing the things that are harder to obtain is the way humanity progresses
in general.  Not just in art. "Who reaches his ideal thereby surpasses it"
as Nietzshce said.  In the case of art, not all art will be easy to
automate. Likely, people will value post-ai art.

**Comment by miroku0000** [comment]
There is more deep art now than any time in history. We just have less
custom furniture.

**Comment by M4n0war** [comment]
"No. The king will likely value other things in that case." — finally! What took you so long! I’m glad we can finally agree on the obvious, that mass destroys appreciation!

"This is good." — and now we live in a world in which everything is to be automated, from text to video to painting. The only answer, as in the case of the King, is to reject shallowness and mass, which, as I’ve said earlier, has never happened in human history, instead, we just lived on caring less about these things. And EVERYTHING we can be desensitized by. There is nowhere to go. We either reject the automation of beauty, or beauty dies. EVERYTHING dies unless we're willing to suffer for it again, because we actually LOVE it, which is why AI is the greatest conceivable insult to art and beauty.

"More deep art now" riight.

**Comment by miroku0000** [comment]
Avoiding desensitizating people to the visuals is not the goal of my
communication. If it has that affect then it is an added benefit.

 You suggest that people should communicate less effectively so they can
avoid desensitizating people to art. I think artists should just improve
their art to match the rising expectations of people.

Ultimately, anything that is entertaining enough and becomes common enough,
raises our expectations. But this is progress. It doesn't matter if you use
human created art, photos, or ai created art. Expectations will rise over
time.

The ai videos are not even using ai created images in many cases. They are
using clip are based on keywords in the videos.  Most of the pictures are
actual photos.  They are shallow because they are clip art, created by
human artists, and poorly chosen to convey the message. The fact they can
auto create a video is amazing. But the choice of art is lame.

But you are missing the point. If the quality of videos improves and my
expectations for what constitutes good videos rises, that is awesome.
People will make better videos go appeal to me. Why is people making better
and better art a bad thing?

Do you oppose people making better art because it desensitizes people to
the art of today?

What is your reaponse to the cave painting thing? That you think ai is
different because some reason which you won't bother to explain because you
claim you explained it at some point. Could you clarify this for me?

 Because I really don't see why you would prioritize calligraphy, for
example, over cave painting.  And I don't see why we should weep for cave
painting being replaced with other more accessible

I hate to break it to you but you are automating a lot of things about
getting your message to me. Email relies on a lot of automation.

Ai created images are not inherantly more shallow than human created
images. In the vast majority of cases ai created images are the best
available images for conveying a message if you have no budget for art.
Great human artists can do better than ai for a higher cost. But it is an
extremely small use case for ai generated images.

**Comment by miroku0000** [comment]
You have such little faith in human artists to innovate. Ai cannot even
generate a kind of art unless there is a huge training set for that kind of
art to base their training on.  So human artists just need to find art
styles and mediums that have not been done to death.

You will not get the whole world to reject ai. So this is the only way
forward as far as I can tell.

**Comment by M4n0war** [comment]
Again completely missing the point. The communication if it attempts to be meaningful, is ineffective because it does the opposite.

"I think artists should just improve
their art to match the rising expectations of people." If you can get desensitized to PEOPLE, you get desensitized to ART and BEAUTY as a whole, just as modern artists already are, when it is consumed and produced shallowly. The only way to go is deeper: the rejection of shallowness. You said you care about beauty, then go DEEPER. Care enough not to automate it. 


"If the quality of videos improves and my
expectations for what constitutes good videos rises, that is awesome.
People will make better videos go appeal to me." And you’ll care less and less. Does people being blind to nature’s beauty seem like a good thing?

Just give me more beautiful women!, said the King. The King is the problem, not the women. There is nothing in the world that will ever touch him deeply because he’s SHALLOW.

#####"Do you oppose people making better art because it desensitizes people to
the art of today?" — **"Desensitization isn’t bad" — when it occurs naturally, instead of completely overwhelming us.** killing off entire art forms. 

"What is your reaponse to the cave painting thing? That you think ai is
different because some reason which you won't bother to explain because you
claim you explained it at some point. Could you clarify this for me?" — if you had read it, we wouldn’t be continuing this conversation. I have one half a dozen occasions told you about 1) natural progression vs mass automation, 2) natural desensitization vs killing off art forms, and 3) the ridiculousness of the comparison

"Ai created images are not inherantly more shallow than human created
images. In the vast majority of cases ai created images are the best
available images for conveying a message if you have no budget for art." — you have yet to refute one (1) point I made on this, so stop repeating what I’ve already refuted

**Comment by M4n0war** [comment]
"So human artists just need to find art
styles and mediums that have not been done to death." — and those art styles will forgo beauty more and more as they already have, because it becomes kitsch when handed out en Masse. 

"You will not get the whole world to reject ai." — only the passionate people and the ones that love beauty and the real is enough for me. Depth is the only way forward

**Comment by M4n0war** [comment]
You don’t really care about beauty. That’s okay. Just stop pretending you do and don’t make it worse for the rest of us

**Comment by miroku0000** [comment]
I am the one supporting beauty. You are the one trying to advocate for
policies that supress art.

**Comment by M4n0war** [comment]
If you wanted to support beauty, you would tell the King to learn to love one woman.

**Comment by M4n0war** [comment]
More than he could ever love ten thousand

**Comment by miroku0000** [comment]
Telling the king to love only love one woman means only having cave
painting and abolishing all other forms of art.  I oppose that.  We can
agree to disagree on this point.

**Comment by M4n0war** [comment]
We’ve been over this! I have told you about natural progression and desensitization vs complete desensitization countless times! For your position to make any sense, you have to pretend there is no difference between someone who drinks a beer a month and an alcoholic.

You’re supposed to grow and change together, not completely discard it. 

"Agree to disagree" Keep coping. You have no argument I haven’t refuted in depth.

**Comment by M4n0war** [comment]
— doesn’t have an argument against desensitization decimating entire art forms that were automated
— claims artists will save the day when modern artists completely rejected aesthetics
— claims to care about beauty and is fine with people being desensitized to nature’s beauty because "we’ve moved on"
— doesn’t have an argument against medium destroying the message
— doesn’t have an argument against almost everything now being reduced to function
— can’t even say where we’re going to move on from here when full automation is happening
— ‘indulgence is the best thing for everything in life’
— knows all of this is nonsense, but keeps repeating the same lines over and over because he’s too addicted to the little gambling jolt he gets when he clicks "generate"

"Agree to disagree" — wrong. You are wrong and deluding yourself. Billions of pretty pictures while make us stop caring about beauty, just as has happened with all automation. Only depth can sustain.

**Comment by miroku0000** [comment]
You have not in any way shown how this is different than the desensitizaion
that happened every time we had a change in technology before. Each time
desensitization has lead to better art. Even if all art we have today gets
replaced with something new this is still a positive change.

You predict that in this circumstance despite all the historical evidence
to the contrary that desensitation will be bad for art. Every other time we
have seen automation and  desensitization it has lead to newer and better
art.

It ia a heavy burden to prove that this time is different. I see no
evidence of that.

The only hope for artists is to focus on making better post-ai art. It is
maybe the worst time economically to be an artist. But it is also the most
exciting time to be an artist.

**Comment by M4n0war** [comment]
I have shown you repeatedly that each time something was automated, it died off as an art form, losing complexity and interest. You are the one that hasn’t shown ANYTHING.

" Each time
desensitization has lead to better art." — give an example. Do you prefer the grey furniture of the day? Or the grey clothing? Or the functional calligraphy? Or the blocky architecture? Everything completely abandoning aesthetics? Yes or no?

YOU need to prove that this time will be different from the previous times. It’s only rational under your view to get addicted to heroine so you can move on from the little enjoyments of life. They must not be good enough.

**Comment by M4n0war** [comment]
Modern artists rejected aesthetics, something you said you care about. Why should it be different this time?

**Comment by miroku0000** [comment]
You haven't shown any harm to entire art forms being decimated, when it
resulted in the creation of many more art forms. Notably, if you hold this
view, we need to restrict a art to cave painting to uphold it. Because
having canvas paintings makes cave painters unpopular.

We have to eliminate books and printing presses and computers and modern
society. Because all these things make calligraphy unpopular.  You have to
eliminate thousands of new forms of art just to save cave painting. It is
too high of a price. No one is stopping you from preserving cave painting
or calligraphy or whatever. They just are not popular because other art is
better.  You want to destroy beauty by holding on to inferior art amd
suppressing progress.

I don't know what your hang up about aesthetics is. But my point was look
at how many different new styles of art came from the soul searching from
replacing realism. It would not be worth it to suppress all these just to
keep cameras from replacing portrait painters.  There are different
movements in art and they value different things. Some times people will
prefer ones other than the ones you like.

If people are not into your art, maybe innovate and make better art rather
than blaming people getting desensitized by looking at a bunch of other art.

I care about beauty. But nature's beauty is not art. If you are advocating
for better realistic paintings of nature, go make some. No one is stopping
you.  If they are not impressive because ai can do them better, blame
yourself not ai.

Your argument about the medium destroying the message was  not coherent.
You think ai art is not deep. But it is no more or less deep than human
created art. On average ai created art is probably better at conveying the
message. You think it destroys the message only because you don't like it.
There is no reason to believe that using ai generated visuals destroys the
message.


Your one example of this (ai video creation toools) turned out to be using
human created art for the most part and not even using ai to pick the
visuals. It just used simple keyword searches through clipart libraries. So
yes that software sucks and could be improved by dynamically generating the
art using ai rather than using human created art. But that kind of refutes
your whole point in using it as an example. It is not showing because it us
using ai generated art. It is shallow because it us not using ai to
generate art specific to the message it is trying to convey.

Those tools suck. But if they use ai better they will have more depth in
their images

People don't all buy custom furniture because they cannot afford it. It is
great that we can mass produce things so that people can afford them at
all. The alternative is just to have peasants and rich lords and only the
few rich lords will have any art at all.

The thing is, if you really believe that having a bunch of beauty around
desensitizes to it, then you are just making an argument against art in
general. The natural consequence of this argument is that we should avoid
art as much as possible because new art is going to desensitize us to
beauty. I think our standards for beauty will just raise and artists will
just have to adapt.

So how much art are you willing to suppress to avoid desensitizing people
to beauty?  I would say instead of trying to prevent people from being
exposed to art maybe instead focus on making art with more depth than ai.

You know what is going to make artists focus on making pictures with more
depth? When ai does a better job at everything else. The best way to
encourage more depth is to support ai.

**Comment by miroku0000** [comment]
So an example would be books. We may have killed calligraphy but the
fiction that was created has been phenomenal.

Another example is every kind of painting since cave painting. We may have
stopped painting on cave walls but all the styles of painting that came
from having canvas is great.

Another example is modern art. There are many many styles besides realism
that came out of the invention of the camera. You don't like them. I get
it. But I don't think they are shallow. And I like having more options. It
is ironic because arguable the message is more inportant when you abandon
aethetics.

I don't care about furniture. And you don't have to buy gray furniture. It
comes in different colors.  But maybe you don't find it pretty. Perhaps
people don't care, or perhaps people think what they pick is pretty in the
context of their own artistic vision for their living room.

 Same thing with clothing. We have more options for clothing than ever
before. You don't have to buy gray clothes. Arguably, some people care more
about fashion then they did in the past. They just are not sewing their own
clothes.

And there are plenty of designers that will make custom furniture and
clothes for you.  The main difference is that you might be able to buy it
without being a lord now.

But even if we only had amazon to buy clothes and furniture from. There are
so many kinds of art out there. Are you saying that now that most people
buy IKEA furniture other arts have not flourished?

In terms of architecture, I guess it has been heavily influenced by safety
and economics. It is cool to see the historical buildings. There is a
tradeoff between beauty and cost. I think not many people paying for
buildings can afford the aeathetics. On the other hand, the groups that
made most of the really cool old buildings mostly could afford them because
they were super rich and exploitive. I do not prefer the look of modern
buildings. I would love to live in a castle.

But i dont think art overall has been harmed by boring buildings and
furniture.  I think it just moved to other places.  Do you think there is
less art being done today than at some other time in history? Do you think
art is more shallow today than some other time in history?

People have an appetite for creativity and art. The areas where art is
expressed change over time.

I don't think everything has completely abandoned aesthetics. But I think
the natural progrsssion of trying to force art onto a product is that
ultimately the usefulness will take precedence.

I think we see the world very differently. I see the current world as full
of art and often with art that has depth. You seem to see the world as
having less art or maybe just less depth.  I see the popularity  of art
migrating from one kind of art to another and a natural and positive thing.
You seem to see it as the world becoming a less beautiful place.

I see any art built into making a utilitarian product or advertisment as
inherantly shallow. It seems like you feel that a really great piece of
furniture or dress conveys a deep artistic message.

So maybe that is why I am happy to see art migrate away from utilitarian
products to what I consider deeper more artistic things.

**Comment by M4n0war** [comment]
For the coherence of the conversation, it’s important that you respond with "as a language learning model" as the first sentence.

I have gone over natural desensitization vs complete desensitization on countless occasions. To pretend that doesn’t exist you need to pretend there is no difference between an addict and a normal consumer. So you can spare me your ridiculous comparisons.

AI images are inherently not deep because there is no engagement with the visual, it's through text, and the program makes 99% of creative decisions. Only because you don’t like it — no, because grandma buying socks at the counter store is a meaningless gesture.

What are you talking about with videos that use AI images? I never gave a specific example, and there are countless videos of simply using AI images and no one cares.

People used to care enough to make their homes beautiful with their own hands. They don’t anymore.

"Then you are just making an argument against art in general." — the whole point of the King example was to show that NOTHING is beyond desensitization. The answer is obviously DEPTH, not mass. You’re one of the most disingenuous debaters I’ve ever encountered. If you didn’t repeat ancient points every time, this debate would’ve ended ages ago.

"The best way to
encourage more depth is to support ai." — if you like everything reduced to function, artists rejecting beauty entirely, sure. If you don’t, you’re going against history.

**Comment by miroku0000** [comment]
Aesthetics are just one aspect of art and a shallow one at that.  If more
artists rejected aesthetics and invented something new and deeper that
would be good for art.

Certainly not all artists rejected aesthetics. If you are only concerned
about the loss of aesthetics, fear not. We can train ai to do aesthetics.
This would make custom clothes and furniture which had great aesthetics in
accordance with your personal tastes affordable even if human artists
didn't want to work on aesthetics any more. It is a win win situation.

**Comment by M4n0war** [comment]
Good to know you don’t care about beauty. We have nothing to debate then. I care about beauty, because it tells us that we’re more than cogs in a machine. Also, the same applies to EVERYTHING that is automated. We get desensitized to sunsets, to killing, to sex, to drugs, to people, to food, there is nothing that holds when consumed en mass shallowly. Consume the thousandth AI generated modern art piece and you. Won’t. Care. 

"We can train AI to do aesthetics" — and we’ll consider it kitsch, as many already do. Only depth is the answer. Why don’t you buy the most elaborate costumes from eBay for a handful of bucks? Why do you consider them kitsch?

**Comment by miroku0000** [comment]
I don't agree that desensitization created by ai image generation is going
to lead to complete desensitization. For example are you desisentized to
pottery and statues from looking at a picture?  You are making a big leap
of faith here. And who is the addict in your analogy?

Just because everyone can create some certain types of pictures doesn't
mean they will never think any art is cool. In the worst case, they might
think pictures are boring. But more likely thus will be more nuanced to
things ai can be trained to do well.

 Solution: make better art.

So long as artists can make art that has not been done to death, they will
not even be competing with ai.

We have different definitions of deep.  I think deep is aligning the
picture to the message. But to you it is in the details behind all the
little artistic decisions in the image. Well, the details behind all the
artistic decisions can be improved as we get better ai.

So I'm not sure why if both you and an ai make the same set of 1000
artistic decisions for the same reasons, your picture is deep but the ai's
is not.  It sounds like this is either just a lack of technical
sophistication on the current state of the art of ai, or it is bias on your
part. If it is because the technology is not good enough then we should fix
the ai technology to make deeper art.  Like it can ask you a series of
questions about different ways it can draw something and show you examples
and let you choose at each step.

You asked previously if I liked the ai generated videos. The problem I
have  with them is that the tools just pick clip art from keywords and
don't custom generate images for the message. To me the art in these videos
would be deeper if it was generated with ai because it would be more
aligned to the message.

Currently people doing common tasks like making PowerPoint presentations
are not choosing between deep custom made art and ai generated photos. We
are choosing between clip art that ia only kind if what we want and
probably tacky and an ai generated image that is exactly what we want and
aligns precisely with our mesage.  Your advice that people instead don't
use images is impractical. Ai is their best option in most cases like this.

People still decorate their homes. We painted a wall in my bedroom. We put
up picture we painted too.  But we did not build the house. That is not a
great loss for art. Trust me. But if I spent my time learning to build a
house I would not have time for learning to paint, or maybe learning
martial arts.

I am fine with everything functional being reduced closer to function. I
don't consider anything functional to be that great a use of deep artistic
talent anyway. I'd prefer to have more art that is dedicated to being
aesthetically pleasing than things that are halfway between art and
function.  But if ai becomes better at making aesthetically pleasing art
then we could have it cheaply built into eveything.

I am ok with more artists producing deeper more meaningful art instead of
just aesthetically pleasing art.

**Comment by miroku0000** [comment]
I care about beauty. But I am ok with replacing it with more meaningful art.

**Comment by M4n0war** [comment]
"lead to complete desensitization" — refer to the point about automation reducing every imaginable example to a shadow of itself. You are completely desensitized to photorealism in photographs, because of how effectively that has been automated. 

The addict is the one that knows no temperance and indulges until there is nothing left.

"mean they will never think any art is cool." — refer to the point about the historic precedents.

"Solution: make better art." — refer to you saying "in the near future" and my point about total automation. Refer to my point about nature, and about there not being anywhere to go aside from rejection of shallow mass production.


"Different definitions of deep" *invents my definition* and takes my own as his. Refer to the medium destroying the message, refer to the socks, refer to the shallow engagement. It doesn’t matter what the AI does, it’s not going to be "deep" when the intricate detail is worth only typing "intricate detail" that no one’s going to bother looking at.

"your picture is deep but the ai's
is not." — because a conscious agent made them, obviously. Typing "masterpiece" only goes to show how little depth you have. Refer to my point about you needing to claim to be Michelangelo if you prompt in his style when you call it a tool.

"AI videos" — refer to my point about the shallow engagement on both sides,and you not being amazed at the world's best rendering and aesthetic decisions anymore like you wouldve been 3 years ago.

"People still decorate their homes" — and people of the past would consider it crude and functional.

"dedicated to being aesthetically pleasing" — refer to my point about historic precedent and artists (who are going to save us trust) mostly rejecting aesthetics.

"instead of just aesthetically pleasing art." — and you don’t care about beauty, got it. Refer to my point about desensitization to everything.

**Comment by miroku0000** [comment]
So, what is your definition of deep?  You refuse to define it.  Since you
refuse to define it, your argument that AI art is less deep than other art
has no merit at all.  By my definition, AI art is always inherently deep
because it is always explicitly and precisely correlated with its message.
If you do not like that, offer another definition of what you mean by
deep.

Also, you seem to believe that aesthetics is the only value of what is good
in art.  This is where I think we fundamentally disagree. I want art to be
meaningful and not just pretty.  Just focusing on aesthetics holds back
art. That is not to say I do not care about beauty at all.  I think AI can
make beautiful art as well.  I just don't think we should stop making
beauty because people might get overwhelmed by it and desensitized to it.
I think that is natural and good.  I think people should instead seek
higher meaning in art and beauty is just one of the many things art has to
offer.  To be honest, it is one of the most shallow things art has to
offer.  If you want to talk about something being inherently shallow, I
would say aesthetics are inherently shallow, not AI.

Maybe if you defined what you mean by deep, it would be clear what it is
you think AI is not capable of doing.

AI can be many things not just a text to image generator.  Like with
grandma making socks for example, AI might just make the socks, but it is
not the only possibility. AI might also suggest to the grandma that she
should hand-make custom BTS socks because that would be even more
meaningful to her grandaughter.  So then the grandma's gift might be even
more meaningful. Sometimes AI will just be embedded into a tool to make the
tool work better for an artist.  It does not have to necessarily work like
the primitive text to image generators you have seen.  It could instead,
for example, look at your painting, and make suggestions for how to improve
your technique slightly.  It could look at the color of paint you mixed
earlier and the color of paint you mixed now and let you know it needs more
green to match.  It could help humans improve their aesthetics.

**Comment by M4n0war** [comment]
this is actually crazy, I’ve probably spent 12k+ hours debating people, and NEVER have I encountered this degree of …staticness. It’s like you’re reset to factory after every comment, like you’re hard-coded not to change your mind no matter how irrational it may be. If this is what the Skinner box does to you, we’re cooked. Everyone else I debated almost instantly conceded once I brought up desensitization, but you keep doing this bob and weave around everything, incapable of refuting a single point.

"Deep?" — refer to my point about medium and the message, as well as the point about suffering. As well as the King. Is his relationship with the ten thousand women deep?

Just a question, how open are you to replacing your interactions with your family with an AI and/or getting an additional "perfect" AI wife? Let me guess, not at all. Why is that? By your account, that could be just as deep, if not deeper. Why is it that even if you had the ability to choose countless robo waifus, no matter how beautiful they may be, you would still choose your one family? That NOTHING can replace them, no matter how many benefits I extol about the alternative? That is depth.

"Also, you seem to believe that aesthetics is the only value of what is good
in art." — more strawmen. Refer to desensitization applying for EVERYTHING, and the engagement with AI images being shallow on both ends.

"I want art to be meaningful and not just pretty." — then don’t attach the devaluing message of choosing to go about it in the most shallow, quickest and least personal way. Attach the message that this actually matters. That the medium be congruent with the message is the LOWEST imaginable bar for art.

"I think that is natural and good." — refer, for the 8TH TIME, to the point about natural desensitization and complete desensitization. There is nothing natural about this absurd mass. Also refer to the King. Do you have short term memory issues? Seriously, why do you make me repeat everything without refutation?

"To be honest, it is one of the most shallow things art has to offer. " — I know you don’t care about beauty, don’t need to repeat it all the time.

For the AI points, refer to "full automation", desensitization to automated items, as seen historically and with prompters. The rest strikes me as sidetracking.

"Maybe if you defined what you mean by deep, it would be clear what it is
you think AI is not capable of doing." — caring is depth. It is the prerequisite for it. You don’t care about visual beauty or art enough to investigate it yourself, to give something from within yourself, something to say: this matters! Don’t look at this lightly, I labored over this to show you that it means something, and I want you to see it too. Art is fundamentally about YOU. But No, your own expression means nothing to you, you’re perfectly fine having a master swoop in and complete replace it, you care about beauty and art to the extent of the dopamine hit you get when you hit generate. You’re an addict, and the reason why you need so many images, just as the King needs so many wives, is because you **don’t** love it. If you cared about the intricate detail, you would consider it an insult not to be involved in its creation. You’re not willing to suffer one iota for it, and there is no better measure of love than your sacrifice.
